src/runtime.js

```diff
@@ -500,7 +500,11 @@ function registerElementClass(global, name, implementationClass, opt_css) {
     };
   }
   // Resolve this extension's Service Promise.
-  getService(global, name, emptyService);
+  registerServiceBuilder(global,
+      name,
+      emptyService,
+      /* opt_factory */ undefined,
+      /* opt_instantiate */ true);
```

Comment:
The intention is that we just register the service, that way if anyone has requested the service promise it will be resolved. For the large majority of extensions, they don't need to have it resolved ever.